src/main/javacc/ASTColumnElement.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.spanner.pgadapter.parsers;
+
+public class ASTColumnElement extends SimpleNode {
```

Comment:
Why do we manually add those AST classes? I found that there are a lot more AST classes referenced in the visitor but they are not here so why we have to treat those AST classes specially?

Reply:
I manually added ones that I wanted to give specific functionality to.